src/main/java/org/mule/extension/http/api/request/builder/QueryParam.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.http.api.request.builder;
+
+import org.mule.runtime.extension.api.annotation.param.Parameter;
+
+/**
+ * Provides a way to configure an HTTP Query Parameter
```

Comment:
This doesn't provide a way to configure query parameters, this represents a QueryParameter